src/parsers/utils.hpp

```diff
@@ -52,6 +80,10 @@ namespace se3
     // else return an empty string that will provoke an error in loadPolyhedronFromResource()
     namespace bf = boost::filesystem;
 
+    // In case the mesh_path is such as "file://path/to/mesh", check for /path/to/mesh and return it if it exists
+    if ( bf::exists( bf::path(urdf_mesh_path.substr(6, urdf_mesh_path.size())) ))
+      return std::string(urdf_mesh_path.substr(6, urdf_mesh_path.size()));
```

Comment:
Thanks for the comment. I agree that the schema is weak. Lets try to improve it.

However, we should not add boost regex in a header file. The compilation timings of Pinocchio are already way to high. 

One solution could be to move the parser in a cpp file and add it to the library. But one of the terminal objective is to have all the Pinocchio code as pure template (with the Scalar type being templated for floating or double precision, or for automatic derivation). So it might be counterproductive to first introduce regex in a cpp file for then later removing it when the cpp file will become a template header hxx file again.

Could we try to find a solution without adding any further dependancies?